pkgs/development/python-modules/Rtree/default.nix

```
@@ -1,4 +1,4 @@
-{ lib, stdenv, buildPythonPackage, fetchPypi, libspatialindex, numpy }:
+{ lib, stdenv, buildPythonPackage, fetchPypi, libspatialindex, numpy, pytestCheckHook }:
```

Can you convert that to one input per line? This improves diffing and merging.

```suggestion
{ lib
, stdenv
, buildPythonPackage
, fetchPypi
, libspatialindex
, numpy
, pytestCheckHook
}:
```